sources/src/main/resources/META-INF/resources/index.html

```diff
@@ -3,1007 +3,729 @@
-                                <i>External approval</i>: If a role requires external approval, then someone from a
-                                specified group
-                                of users outside your peers must approve your access request before it takes effect.
-                            </li>
-                        </ul>
-                        </p>
-                    </div>
+                <div class="mdc-linear-progress__bar mdc-linear-progress__secondary-bar">
+                  <span class="mdc-linear-progress__bar-inner"></span>
                 </div>
+              </div>
             </div>
-            <div class="mdl-grid center-items">
-                <div class="mdl-card mdl-shadow--2dp">
-                    <div class="mdl-card__title">
-                        <h2 class="mdl-card__title-text">Grant just-in-time access</h2>
-                    </div>
-                    <div class="mdl-card__supporting-text">
-                        <p>
-                            As an administrator, you use IAM role bindings to decide which projects and roles users
-                            can request access to. To mark a role binding as <i>eligible</i>, you add a special
-                            <a href="https://cloud.google.com/iam/docs/conditions-overview" target="_blank">IAM
-                                condition</a>:
-
-                        <ul>
-                            <li>
-                                <i>Self-approval</i>: To allow a user or group to activate a role without approval, add
-                                the
-                                following IAM condition to the IAM binding:
-                                <pre><code class="language-cel">has({}.jitAccessConstraint)</code></pre>
-                            </li>
-                            <li>
-                                <i>Peer approval</i>: To allow a user or group to activate a role if at least one peer
-                                approves their request, add the following IAM condition to the IAM binding:
-                                <pre><code class="language-cel">has({}.multiPartyApprovalConstraint)</code></pre>
-
-                                Peers are users that have been granted the same role binding on the same project.
-                            </li>
-                            <li>
-                                <i>External approval</i>: To allow a user or group to activate a role if at least one
-                                external
-                                reviewer approves their request, add the following IAM condition to the IAM binding:
-                                <pre><code class="language-cel">has({}.externalApprovalConstraint)</code></pre>
-
-                                External approvers are users that have been granted the same IAM binding with the
-                                following
-                                IAM condition:
```

Comment:
Has all the information been moved somewhere?

Reply:
No. I think this part has been removed in favor of the central documentation. I will update the central documentation accordingly.